build.gradle.kts

```diff
@@ -440,10 +440,6 @@ tasks.register("goPortablePreCommit") {
   dependsOn(":sdks:go:test:ulrValidatesRunner")
 }
 
-tasks.register("goPostCommitDataflowARM") {
```

Comment:
I think i suggested to remove it before offline, but feel free to keep this, looks like it's a convention that other `go` tests follow even if there is dependency only on one target.

Reply:
@lostluck thoughts?

Reply:
I have no opinion on gradle. Follow the conventions that exist.